sdcm/sct_runner.py

```diff
@@ -67,6 +67,9 @@ def install_prereqs(self, public_ip):
         LOGGER.info("Installing required packages...")
         prereqs_script = dedent(f"""
             echo "fs.aio-max-nr = 65536" >> /etc/sysctl.conf
+            echo "ubuntu soft nofile 4096" >> /etc/security/limits.conf
```

Comment:
why not setting `fs.file-max` in `/etc/sysctl.conf`, it is systemwide...

Reply:
I did it per user because of on such way I can set just soft limit.
system-wide limit in /etc/sysctl.conf set "fs.file-max = " higher than user-limit. I didn't see that it impact on soft. It will increase “total” number of files that can remain open system-wide